routes/users.routes.js

```diff
@@ -58,4 +59,39 @@ router.post("/signup", validateEmpty, validPwd, (req, res) => {
   });
 });
 
+
+/* GET/ signin */
+router.get("/signin", (req, res, next) => {
+  res.render("users/signin.hbs");
+});
+
+/* POST/ singin */
+router.post("/signin", (req, res, next) => {
```

Comment:
Wish: the validations are missing. We should validate here that the user filled out the fields. You could just use the validator from line 20

```suggestion
router.post("/signin", validateEmpty, (req, res, next) => {
```

Reply:
Ops! fixed! :D